### ISSUE TITLE
Failed deploy as-is

### ISSUE THREAD

**Issue by wilmeragsgh** [render-examples/mysql#1]
When building it throws an error like this:

```sh
https://repo.mysql.com/yum/mysql-5.7-community/docker/x86_64/repodata/repomd.xml: [Errno 14] HTTP Error 404 - Not Found
...
```
At the line where it should be installing tzdata in the dockerfile

**Comment by wilmeragsgh** [comment]
I created a fork for the repo and omit the tzdata installation and it worked, however, I'm not sure this is the proper way to workaround this.

**Comment by jeebay** [comment]
@wilmeragsgh Running into this too... I'm fairly new to Docker and Linux things but I from what I can tell, I think omitting tzdata should be fine. I built a container with the tzdata install omitted and then tried to run the command manually. It fails because there is a bad yum repo, `MySQL 8.0 Server Minimal`. I think the baseurl for it is misconfigured resulting in a 404 and yum quits once it hits that error. I uninstalled the repo with `yum-config-manager --disable "MySQL 8.0 Server Minimal"` then tried `yum install -y tzdata` which succeeded but with the message

```
sh-4.2# yum install -y tzdata
Loaded plugins: ovl
ol7_UEKR6                                                                                          | 2.5 kB  00:00:00     
(1/2): ol7_UEKR6/x86_64/updateinfo                                                                 | 126 kB  00:00:00     
(2/2): ol7_UEKR6/x86_64/primary_db                                                                 |  15 MB  00:00:01     
Package tzdata-2021a-1.el7.noarch already installed and latest version
Nothing to do
```

I assume this means that the underlying image is already including the most up-to-date `tzdata` database so we can remove this line?

**Comment by jeebay** [comment]
@wilmeragsgh I spent a decent portion of the day looking into this but looks like it was fixed in #2 which was merged to master earlier today! If you pull or update your fork you should be fine.

**Comment by wilmeragsgh** [comment]
Thanks @jeebay ! Yes, just saw the commit closing this then, thanks again for commenting on it too.
